### PR TITLE
added property accessor in columndefinitions for stores

### PR DESCRIPTION
**Pull request by remyyounes**
Allows for data to be displayed and filter on the correct field:
 - introduces a `columnDefinitions` property `textField:func` that defines how to access a key on an object.
 - introduces a `config` property  `columnDefinition` used at store creation.

### CODE REVIEW

**Comment by yoyis3000** [comment]
+1

File: lib/stores/LokiStore.js [hunk 1]
```diff
@@ -359,13 +360,19 @@ function _buildSearch(query) {
   };
 }
 
+function getPropAccessor(key) {
+  let def = _.filter(this.columnDefinitions, (def) => def.key === key);
+  if (def.length) return def[0].textField;
```
Comment (michaelghinrichs) [inline lib/stores/LokiStore.js]: brackets? 
```js
 if (def.length) { return def[0].textField; }
```

**Comment by michaelghinrichs** [comment]
comment is nb, +1

**Comment by brock-noah** [comment]
+1

**Comment by chrisarevir** [comment]
+1, brackets would be nice

**Comment by dannyphillips** [comment]
+1

**Comment by remyyounes** [comment]
Comments have been addressed

**Comment by michaelghinrichs** [comment]
this good to merge?

**Pull request merged**